eng/common/powershell/create-tags-and-git-release.ps1

```diff
@@ -109,7 +109,7 @@ function ParseMavenPackage($pkg, $workingDirectory) {
     return $null
```

Comment:
For this file I'm not entirely sure having a powershell directory is all that interesting. I would probably put this as well as the Extract-ReleaseNotes.ps1 script under a `eng/common/scripts` directory. It is find if we end up with other type of scripts in there. It will just be a directory for lose one-off scripts.